packages/account/src/Components/poa/common-mistakes/common-mistake-examples.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import IcDocumentNameMismatch from 'Assets/ic-document-name-mismatch.svg';
+import IcDocumentAddressMismatch from 'Assets/ic-document-address-mismatch.svg';
+import IcOldIssuedDocument from 'Assets/ic-old-issued-document.svg';
+import IcBlurryDocument from 'Assets/ic-blurry-document.svg';
+import IcCroppedDocument from 'Assets/ic-cropped-document.svg';
+import IcEnvelope from 'Assets/ic-envelop.svg';
+import IcErrorBadge from 'Assets/ic-error-badge.svg';
```

Comment:
@likhith-deriv can't we use `require` to import these files and place them in an array?